src/sentry/static/sentry/app/components/events/interfaces/frame.jsx

```diff
@@ -93,6 +93,19 @@ const Frame = createReactClass({
     };
   },
 
+  componentDidMount() {
+    //Need to update if the SentryAppComponentsStore changes
```

Comment:
could you maybe add a bit more to the comment about what we talked about IRL? Just like about how it's possible for this component (`frame.jsx`) to mount before `fetchData` in `groupEventDetails` or wherever has finished.